### PR TITLE
Update sane-pdf-report script with new docker image

### PR DESCRIPTION
**Pull request by yanyo87**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

> https://github.com/demisto/sane-reports/pull/165
> https://github.com/demisto/sane-reports/pull/156
> https://github.com/demisto/sane-reports/pull/147
> https://github.com/demisto/sane-reports/pull/137
> https://github.com/demisto/sane-reports/pull/152
> https://github.com/demisto/sane-reports/pull/130
> https://github.com/demisto/sane-reports/pull/129
> https://github.com/demisto/sane-reports/pull/133

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Update `SanePdfReport` to latest docker image with latest `sane-report`.
which includes 6.2 reports and dashboard functionalities changes & bug fixes.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Base/ReleaseNotes/1_12_4.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#### Scripts
+##### SanePdfReports
+- Multiple bug fixes
```
Comment (bakatzir) [inline Packs/Base/ReleaseNotes/1_12_4.md:3]: If you do not explain what was wrong and how it was fixed, keeping this line is worse than deleting it.

Comment (roysagi) [reply Packs/Base/ReleaseNotes/1_12_4.md:3]: @yanyo87 - You can use the docs https://xsoar.pan.dev/docs/documentation/release-notes#bug-fixes-examples for examples on how bug fix RN should look like.

Comment (yanyo87) [reply Packs/Base/ReleaseNotes/1_12_4.md:3]: i'm familiar with that, 
but we have more than 50 prs 
image

**Comment by yanyo87** [comment]
@kirbles19 can you review this

File: Packs/Base/ReleaseNotes/1_12_4.md (hunk 1 above)
Comment (dorsha) [reply Packs/Base/ReleaseNotes/1_12_4.md]: We should state that the new docker version includes the new version of sane-reports, which includes new reports and dashboarding functionalities / bug fixes.
We should obviously not write all changes, and state only the major ones with a reference to the sane-reports repo (which is an open source repository).

**Comment by yanyo87** [comment]
@richardbluestone \ @yaron-libman do we have another blocker ?

**Pull request merged**